### ISSUE TITLE
Suggestion: verbosity for commands

### ISSUE THREAD

**Issue by NyaomiDEV** [nic96/mcpelauncher-linux_display_events_mod#1]
Can you please add commands verbosity?
It should be like this
"Player issued command: /time set 0"
"[Player: set time to 0]"

**Comment by nic96** [comment]
Good idea, I might look into how I could implement this when I have time.
